extensions/yaml_support/labgraph_monitor/server/lg_monitor_server.py

```diff
@@ -70,14 +53,12 @@ def setup(self) -> None:
             self.WS_SERVER_NODE.configure(wsapi_server_config)
 
         def connections(self) -> lg.Connections:
-            return (
-                # Test Node Connections
-                (self.NOISE_GENERATOR.OUTPUT, self.SERIALIZER.INPUT),
-                (self.SERIALIZER.TOPIC, self.WS_SERVER_NODE.topic),)
+            local = ((self.SERIALIZER.TOPIC, self.WS_SERVER_NODE.topic),)
+            tuples = local + graph.connections()
+            return tuples
 
         def process_modules(self) -> Tuple[lg.Module, ...]:
-            return (
-                # Test Node Process
-                self.NOISE_GENERATOR, self.SERIALIZER, self.WS_SERVER_NODE)
+            tuples = ((self.SERIALIZER, self.WS_SERVER_NODE)) + graph.process_modules()
+            return tuples
 
-    lg.run(WSSenderNode)
+    lg.run(WSSenderNode, graph.__streams__)
```

Comment:
4. Since I'm setting up the `Demo` graph separately, LabGraph does not have access to the streams it created for `Demo` when dealing with `WSSenderNode`. So I pass it along to update `WSSenderNode`'s streams later.